CoEpi/ble/Central.swift

```diff
@@ -11,15 +11,21 @@ struct DetectedPeripheral {
     let name: String?
 }
 
+protocol CentralDelegate: class {
+    func onDiscovered(peripheral: CBPeripheral)
+    func onCentralContact(_ contact: CEN)
+}
 
-protocol Central {
+protocol CentralReactive {
     var discovery: PublishRelay<DetectedPeripheral> { get }
     var centralContactReceived: PublishRelay<ContactOld> { get }
+    var delegate: CentralDelegate? {get set}
```

Comment:
Why will be support both delegate and rx? This leads to a lot of confusion and is error prone. Let's stick to one or the other. From the last chats it seems we agreed on RX.